daemon/src/identifier.rs

```diff
@@ -0,0 +1,113 @@
+// Copyright © 2019-2020 The Radicle Foundation <hello@radicle.foundation>
```

Comment:
Was this left behind to live in API?

Reply:
Just removed it :)

Reply:
https://github.com/radicle-dev/radicle-link/pull/576/commits/11f2aa8a81508879e1b20390d91e3a8281a8ae18